pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -243,3 +247,17 @@ func (c *Cluster) getPodLabels(daemonName string) map[string]string {
 	labels["instance"] = daemonName
 	return labels
 }
+
+func (c *Cluster) getEnv() []v1.EnvVar {
```

Comment:
Added this function so that its easy to get env vars required for multiple containers.